Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM postgres:9.6.21-alpine
+
+RUN apk add build-base
+
+COPY . /usr/bin/pgvector
+WORKDIR /usr/bin/pgvector
+RUN make && \
+	make install
```

Comment:
I glance over this thread, https://github.com/ankane/pgvector/issues/3, but I'm not sure what would be the best set of flags here.

Reply:
Just pushed a change to allow for `make OPTFLAGS=""` for maximum compatibility